.github/workflows/check.yml

```
@@ -69,7 +68,7 @@ jobs:
           fi
       
       - name: Static Analyzer Reports Uploading
-        if: steps.analyzer-output.outcome == 'failure'
+        if: ${{ failure() }}
```

```suggestion
        if: failure()
```
should be enough, brackets are necessary if you, for example, want to negate it: `if: ${{ !failure() }}`

---

Look at 3d3e878bd080519710493ffc4751c128993c3b47

---

ok, now it makes sense 👍